main/res/values/strings.xml

```diff
@@ -2427,4 +2427,39 @@
 
     <string name="touch_again_to_exit">Please touch again to exit</string>
 
+    <!-- Smileys GC / OC -->
+    <string name="smiley_smile">smile</string>
+    <string name="smiley_big_smile">big smile</string>
+    <string name="smiley_cool">cool</string>
+    <string name="smiley_blush">blush</string>
+    <string name="smiley_tongue">tongue</string>
+    <string name="smiley_evil">evil</string>
+    <string name="smiley_wink">wink</string>
+    <string name="smiley_clown">clown</string>
+    <string name="smiley_blackeye">black eye</string>
+    <string name="smiley_8ball">eight ball</string>
+    <string name="smiley_sad">sad</string>
+    <string name="smiley_shy">shy</string>
+    <string name="smiley_shocked">shocked</string>
+    <string name="smiley_angry">angry</string>
+    <string name="smiley_dead">dead</string>
+    <string name="smiley_sleepy">sleepy</string>
+    <string name="smiley_kisses">kisses</string>
+    <string name="smiley_approve">approve</string>
+    <string name="smiley_disapprove">disapprove</string>
+    <string name="smiley_question">question</string>
+    <string name="smiley_cry">cry</string>
+    <string name="smiley_embarassed">embarassed</string>
+    <string name="smiley_footinmouth">foot in mouth</string>
+    <string name="smiley_frown">frown</string>
+    <string name="smiley_innocent">innocent</string>
+    <string name="smiley_kiss">kiss</string>
+    <string name="smiley_laughing">laughing</string>
+    <string name="smiley_money">money</string>
+    <string name="smiley_sealed">sealed</string>
+    <string name="smiley_surprised">surprised</string>
+    <string name="smiley_tongueout">tongue out</string>
+    <string name="smiley_undecided">undecided</string>
+    <string name="smiley_yell">yell</string>
+
```

Comment:
Not being an expert, but shouldn't such kind of terms start with a capital letter in that single word use case?